save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -24,10 +32,24 @@ class Save(
 
         val plugins: List<Plugin> = emptyList()  // todo: discover plugins (from configuration blocks in TestSuiteConfig?)
         logInfo("Discovered plugins: $plugins")
-        plugins.forEach {
-            logInfo("Execute plugin: ${it::class.simpleName}")
-            it.execute(saveProperties, testConfig)
-            logInfo("${it::class.simpleName} successfully executed!")
+        plugins.forEach { plugin ->
+            logInfo("Execute plugin: ${plugin::class.simpleName}")
+            plugin.execute(saveProperties, testConfig).forEach(this::handleResult)
+            logInfo("${plugin::class.simpleName} successfully executed!")
+        }
+    }
+
+    @Suppress("WHEN_WITHOUT_ELSE")  // TestResult is a sealed class
+    private fun handleResult(testResult: TestResult) {
+        when (testResult.status) {
+            is Success -> logDebug("Test on resources [${testResult.resources}] has completed successfully")
+            is Failure -> logWarn("Test on resources [${testResult.resources}] has failed: ${testResult.status.reason}")
+            is Ignored -> logWarn("Test on resources [${testResult.resources}] has been ignored: ${testResult.status.reason}")
+            is Crash -> logError("Test on resources [${testResult.resources}] has crashed: ${testResult.status.throwable.message}." +
+                    "Please report an issue at https://github.com/cqfn/save")
+        }
+        if (saveProperties.debug == true) {
```

Comment:
Yes, makes sense. Also, we should probably change `logDebug` to accept lazy messages in future (like `fun logDebug(() -> String)` so that the strings are created only when needed.